doc/source/doc-actions/examples/doc-changelog-basic.yml

```diff
@@ -20,3 +20,4 @@ changelog-fragment:
     - uses: ansys/actions/doc-changelog@{{ version }}
       with:
         token: ${{ '{{ secrets.PYANSYS_CI_BOT_TOKEN }}' }}
+        # use-labels: true  # uncomment this line to use labels instead of conventional commits
```

Comment:
I think we should allow for backwards compatibility until next version and change the logic... We should implement both ``use-conventional-commits`` and ``use-labels`` and ``use-labels`` should default to true, whereas ``use-conventional-commits`` should default to false - temporarily.

Next major release we change the behavior. But this way we can live with a minor release and check how this works without breaking everybody's workflows (in case they don't use conventional commits)